vscode-dotnet-runtime-library/src/Utils/ErrorHandler.ts

```diff
@@ -62,11 +64,14 @@ export async function callWithErrorHandling<T>(callback: () => T, context: IIssu
     catch (caughtError)
     {
         const error = caughtError as Error;
-        context.eventStream.post(isAcquisitionError ?
-            new DotnetCommandFailed(error, context.commandName, getInstallKeyFromContext(acquireContext?.acquisitionContext)) :
-            // The output observer will keep track of installs and we don't want a non-install failure to make it think it should -=1 from the no. of installs
-            new DotnetNotInstallRelatedCommandFailed(error, context.commandName)
-        );
+        if(!(error instanceof DotnetInstallExpectedAbort) && error && error.constructor && error.constructor.name !== 'UserCancelledError')

```

Comment:
I really like how new error messages are added. It adds new error types and reduce the number of irrelevant error posts for the user. Nit: I wonder if it is easier to write this way
```suggestion
        if(!(error instanceof DotnetInstallExpectedAbort) && !(error instanceof UserCancelledError))
```